grafana/README.md

```diff
@@ -0,0 +1,42 @@
+# Prometheus & Grafana for Nessie
+
+## Dev installation for local experiments
+
+These steps are mainly for setting up Prometheus/Grafana/Nessie locally.
+
+* Install Minikube as described in https://minikube.sigs.k8s.io/docs/start/
+* Install Helm as described in https://helm.sh/docs/intro/install/
```

Comment:
maybe also link https://kubernetes.io/docs/tasks/tools/ for `kubectl` if we act like this is for users starting from 0 (using the kubectl alias from minikube did not work with the cmds below).

Reply:
done